packages/hydrogen/src/cart/CartForm.test.tsx

```diff
@@ -240,6 +240,12 @@ describe('getFormInput', () => {
         ],
       },
     });
+
+    if (result.action !== 'LinesAdd') {
+      // Just to narrow down the type
+      throw new Error('Wrong action');
+    }
+
     expectTypeOf(result.inputs.lines).toEqualTypeOf<CartLineInput[]>;
```

Comment:
@wizardlyhel it makes sense we need narrowing the type here, right? I'm not sure how this was passing before.